_partials/cartBox/item_option_quantity.php

```diff
@@ -0,0 +1,36 @@
+<?php foreach ($optionValues as $optionValue) { ?>
+    <?php
+    $optionIndex = $optionValue->menu_option_value_id;
```

Comment:
Can you use an input group here with - + and align to the left to match other options?

Reply:
Yes, I can do, but I don't think it works as well layout wise. It takes up a lot of space and looks odd when you have other option types in the same popup.

See screens attached to show what I mean.

Let me know what you think is best!
<img width="539" alt="Screenshot 2020-06-21 at 18 55 32" src="https://user-images.githubusercontent.com/51899/85231635-eabd0300-b3f0-11ea-82cc-52d455ca4b36.png">
<img width="502" alt="Screenshot 2020-06-21 at 18 55 00" src="https://user-images.githubusercontent.com/51899/85231636-ec86c680-b3f0-11ea-8631-972272fea6a0.png">

Reply:
How about reducing the width of the input without the - +?

Reply:
Yeah thats done and pushed.
Can you rebuild app.css - it errors for me as I dont seem to have some dependencies.

Reply:
Thank you and will do.